src/app/devices/deviceIdentity/components/deviceContentNav.tsx

```diff
@@ -79,7 +79,8 @@ export const DeviceContentNavComponent: React.FC<DeviceContentNavProps> =  (prop
     const groups: INavLinkGroup[] = [{ links: navLinks }];
 
     React.useEffect(() => {
-        const currentRoute = navItems.find(nav => pathname.includes(nav));
+        const foundRoutes = navItems.filter(nav => pathname.includes(nav));
+        const currentRoute = foundRoutes && foundRoutes.length > 0 ? foundRoutes[foundRoutes.length - 1] : ''; // because PnP has its own 'events' route.
```

Comment:
Very minor and unnecessary suggestion to improve readability (I `believe` these evaluate the same):
```suggestion
        const currentRoute = foundRoutes?.[foundRoutes.length - 1] ?? ''; // because PnP has its own 'events' route.
```